ODIN_II/SRC/netlist_create_from_ast.cpp

```diff
@@ -3354,6 +3354,17 @@ void terminate_continuous_assignment(ast_node_t* node, signal_list_t* assignment
                 warning_message(NETLIST, node->loc,
                                 "Assignment (%s) is missing driver\n", pin->name);
             } else {
+                /* For the check - Wire cannot be on left in always block */
```

Comment:
[HINT] Instead of checking for ALWAYS type, go for the non-blocking statement since 1. it is more general 2. it must happen in ALWAYS block.